### DISCUSSION TITLE
My idea :  Have a webhook that publishes the backup file

### DISCUSSION THREAD

**Discussion by LarsLarsLars** [mbround18/valheim-docker#394] [Ideas]
Hey Michael, first thank you for the container. We had tons of fun between friends!

My idea :  Have a webhook that publishes the backup files so any player can download them and run them locally.

Thanks Lars

**Comment by mbround18** [comment]
Huh @LarsLarsLars, thats a unique idea, the files themselves are roughly 30mb in size, where were you thinking the files would get posted to? 
I could add an S3 uploader option? or perhaps an FTP upload option?

**Reply by spannerman79** [reply]
Many storage options are S3 compatable.

For example;

- [Vultr Object Sotrage](https://www.vultr.com/docs/vultr-object-storage/)
- [BlackBlaze](https://www.backblaze.com/b2/docs/s3_compatible_api.html)
- Amazon S3 (obviously 😛 )


If the bucket is setup correctly it can be accessed via http/https.  But that (enable bucket public access) is outside the scope of this repo obviously.

_On a personal note: I use Blackblaze as part of my 3-2-1 backup policy for my NAS.  Much better priced then either Vultr or Amazon's S3_

------

The backup that is shared should only really contain the world itself, so whatever is defined in `WORLD` var.  The four files to backup to share to a web accessable S3 bucket would be;

- `<WORLD>.db`
- `<WORLD>..db.old`
- `<WORLD>.fwl`
- `<WORLD>.fwl.old`
